.gitlab-ci.yml

```diff
@@ -567,9 +569,14 @@ build-rust-doc:
     - buildah push --format=v2s2 "$IMAGE_NAME:latest"
   after_script:
     - buildah logout "$IMAGE_NAME"
-    # pass artifacts to the trigger-simnet job
+    # pass artifacts to the trigger-simnet job 
+    - echo "IMAGE_NAME is ${IMAGE_NAME}"
     - echo "IMAGE_NAME=${IMAGE_NAME}" > ./artifacts/$PRODUCT/build.env
```

Comment:
We can use `tee -a` to get rid of double `echo` here but that's a nitpick, nvm.

Reply:
good tip, I will use it :)